<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataGroupBox</class>
 <widget class="QGroupBox" name="MetadataGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>394</height>
   </rect>
  </property>
  <property name="title">
   <string>Metadata Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="metadataCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>EXIF, IPTC and XMP supported</string>
     </property>
     <property name="text">
      <string>Enable metadata support</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="saveMetadataCheckBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Save also metadata</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="thumbUpdateCheckBox">
     <property name="toolTip">
      <string comment="When image changed SIR'll save current thumbnail in Exif metadata"/>
     </property>
     <property name="text">
      <string>Update thumbnail</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="thumbRotateCheckBox">
     <property name="text">
      <string>Apply Exif Orientation tag to thumbnail</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="rotateGroupBox">
     <property name="title">
      <string>When image is rotated</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rotateRadioButton">
        <property name="text">
         <string>rotate seriously</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="exifOrientationRadioButton">
        <property name="text">
         <string>save orientation in Exif</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="exifArtistCheckBox">
     <property name="title">
      <string>Overwrite Exif Artist</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="HistoryComboBox" name="exifArtistComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="exifCopyrightCheckBox">
     <property name="title">
      <string>Overwrite Exif Copyright</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="HistoryComboBox" name="exifCopyrightComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="exifUserCommentCheckBox">
     <property name="title">
      <string>Overwrite Exif User Comment</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="HistoryComboBox" name="exifUserCommentComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/historycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
